src/parse/asp/parser.go

```diff
@@ -73,6 +78,18 @@ func (p *Parser) MustLoadBuiltins(filename string, contents, encoded []byte) {
 	}
 }
 
+// GetAllBuiltins returns all the builtin rules.
+// *Assuming Parser.LoadBuiltins or Parser.MustLoadBuiltins* has been called
+func (p *Parser) GetAllBuiltins() map[string][]byte {
```

Comment:
Let's do this for now... but I think in the longer term I'd like to get away from using the builtin stuff on the parser for a more generic interface. These accessors feel like they're exposing internals more than I'd like.